Tests/DatadogTests/Matchers/SpanMatcher.swift

```diff
@@ -18,6 +20,9 @@ extension String: AllowedSpanAttributeValue {}
 extension UInt64: AllowedSpanAttributeValue {}
 extension Int: AllowedSpanAttributeValue {}
 
+// Only numeric values are allowed for `span.ç_dd.*`.
```

Comment:
Is that supposed to be `ç_dd` or just `_dd`

Reply:
Indeed thanks.